worlds/terraria/docs/setup_en.md

```diff
@@ -7,16 +7,28 @@ and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Stea
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open TModLoader
+3. Go to `Workshop > Manage Mods` and enable the Archipelago mod
+   - If TModLoader states that you need version 1.4.3, follow the following steps
+     1. Close TModLoader
+     2. Right-Click TModLoader in Steam and select `Properties`
+     3. Navigate to `Betas -> Beta Participation`
+     4. Select `1.4.3-legacy - Legacy - Stable tModLoader for Terraria 1.4.3`
+     5. Update TModLoader through Steam
+     6. Open TModLoader and navigate back to the **Manage Mods** menu
+4. TModLoader will say that is needs to refresh; exit of this menu, and it will do this automatically
+5. Once TModLoader finishes loading, the Archipelago mod is finished installing you can now 
+[connect to an Archipelago game](#joining-an-archipelago-game-in-terraria).
```

Comment:
```suggestion
4. TModLoader will say that it needs to refresh; exit this menu, and it will do this automatically
5. Once TModLoader finishes loading, the Archipelago mod is finished installing and you can now 
[connect to an Archipelago game](#joining-an-archipelago-game-in-terraria).
```